utils/loggers/__init__.py

```diff
@@ -240,21 +380,26 @@ class GenericLogger:
         include:         loggers to include
     """
 
-    def __init__(self, opt, console_logger, include=('tb', 'wandb')):
+    def __init__(self, opt, console_logger, include=("tb", "wandb")):
```

Comment:
When is the `GenericLogger` used? Should we update it to call Comet?

Reply:
Hmm I'm not sure about this one. Will have to check.